src/components/Button.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import './button.css';
+
+interface ButtonProps {
+  /**
+   * Contents of a `<Button>` can be a mix of strings and JSX elements,
+   * and will be flex-aligned.
+   */
+  children: JSX.Element | JSX.Element[] | string;
+  color?: 'gray' | 'blue' | 'red';
+  /**
+   * Makes the button dark with light text.
+   */
+  inverted?: boolean;
+  /**
+   * Adds a visible border to the button.
+   */
+  bordered?: boolean;
+  /**
+   * Adds a notification disc to the button.
+   */
+  notification?: boolean;
```

Comment:
Interesting! Do you have any intended uses for notifications?

If not, might be better to leave it out - less complexity, and less code to maintain!

Reply:
I do indeed have a use, although "notification" was a clumsy word for the semantic meaning of indicating the button is "in use":

<img width="183" alt="button-example" src="https://user-images.githubusercontent.com/4731/224429631-21f31bb5-55cb-44ea-8fcc-23e1972519a9.png">

(Note that I changed the design during implementation, because I kind of prefer the corner layout since it doesn't affect the size of the button. Open to change again if necessary.)

Semantically, "enabled" or "active" are both more accurate. My concern about both of _those_ words is that they already have semantic meaning to buttons: "enabled" is the opposite of "disabled", which is a native `<button>` property, and "active" is already a `<button>` property (at least in CSS, e.g. `button:active`) that means "currently being clicked". 😣

Open to suggested terminology changes here! Maybe just… `inUse` given that's what I naturally said above?

Reply:
Actually I'll stop painting the bike shed. We can rename it if we stumble over it in the future!

Reply:
"Notification" seems like a fine term! Although I did appreciate the "blip" working title you mentioned. :smile: 

I was less concerned with the name and more concerned about including unused functionality, but you have a plan so I'm no longer concerned - thanks for the explanation!